ygot/render.go

```diff
@@ -1256,8 +1331,20 @@ func structJSON(s GoStruct, parentMod string, args jsonOutputConfig) (map[string
 // and float64 values are represented as strings.
 func writeIETFScalarJSON(i interface{}) interface{} {
 	switch reflect.ValueOf(i).Kind() {
-	case reflect.Uint64, reflect.Int64, reflect.Float64:
+	case reflect.Float64:
 		return fmt.Sprintf("%v", i)
+	case reflect.Int64:
+		if val, ok := i.(int64); ok {
+			return strconv.FormatInt(int64(val), 10)
+		}
+
+		return fmt.Sprintf("%d", i)
+	case reflect.Uint64:
+		if val, ok := i.(uint64); ok {
+			return strconv.FormatUint(uint64(val), 10)
+		}
+
+		return fmt.Sprintf("%d", i)
 	}
```

Comment:
As suggested below it would be nice if the cases here can be updated and the cases below removed in favour of this to benefit all users of this function.